src/main/java/eu/dissco/backend/repository/MasJobRecordRepository.java

```diff
@@ -101,6 +102,17 @@ public void markMasJobRecordsAsFailed(List<UUID> ids) {
         .execute();
   }
 
+  public void markMasJobRecordAsRunning(String creatorId, UUID masJobId) throws NotFoundException {
+    var updated = context.update(MAS_JOB_RECORD)
+        .set(MAS_JOB_RECORD.STATE, AnnotationState.RUNNING.getState())
+        .where(MAS_JOB_RECORD.JOB_ID.eq(masJobId))
+        .and(MAS_JOB_RECORD.CREATOR_ID.eq(creatorId))
+        .execute();
+    if (updated == 0){
```

Comment:
I would probably do this in the service and only keep the query in the repository